.github/workflows/common-flow-host.yml

```diff
@@ -50,22 +59,44 @@ jobs:
         run: |
           export GITHUB_TOKEN="${{ secrets.GITHUB_TOKEN }}"
           ./deps/readies/bin/getredis --with-github-token
-      - name: Build
-        run: |
-          make SAN=${{ inputs.san }}
-          make COORD=1 SAN=${{ inputs.san }}
-      - name: Unit tests
-        run: |
-          make unit-tests LOG=1 CLEAR_LOGS=0 SAN=${{ inputs.san }}
-      - name: Flow tests
+
+      - name: Decide if to continue after failure
+        id: continue-on-error
         run: |
-          make pytest QUICK=1 LOG=1 CLEAR_LOGS=0 SAN=${{ inputs.san }}
+          if [[ -n "${{ inputs.san }}" ]]; then
+            echo "decision=true" >> $GITHUB_OUTPUT
+          else
+            echo "decision=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Build Standalone
+        if: ${{ inputs.standalone }}
+        run: make SAN=${{ inputs.san }}
+      - name: Build Coordinator
+        if: ${{ inputs.coordinator }}
+        run: make COORD=1 SAN=${{ inputs.san }}
+
+      - name: Unit tests (standalone)
+        id: test1
+        if: ${{ inputs.standalone }}
+        continue-on-error: ${{ steps.continue-on-error.outputs.decision }}
```

Comment:
We can. At first I wasn’t sure what will be the conditions for continuing, and I think it will be easier to change the conditions this way, but for now it will be enought